x-pack/plugins/actions/server/lib/action_executor.ts

```diff
@@ -105,7 +105,7 @@ export class ActionExecutor {
         name: `execute_action`,
         type: 'actions',
         labels: {
-          actionId,
+          actions_action_id: actionId,
```

Comment:
Following our new terminology, I think this should be `actions_connector_id` and `actions_connector_type_id`

Reply:
Thanks, updated the labels w/ new terminology!